sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -38,152 +38,152 @@
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-management</artifactId>
-        <version>1.11.3</version>
+        <version>1.11.4</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-gson</artifactId>
-        <version>1.2.2</version>
+        <version>1.2.3</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-jackson</artifactId>
-        <version>1.4.2</version>
+        <version>1.4.3</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-cosmos</artifactId>
-        <version>4.48.0</version>
+        <version>4.49.0</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-cosmos-encryption</artifactId>
-        <version>2.3.0</version>
+        <version>2.4.0</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-data-appconfiguration</artifactId>
-        <version>1.4.7</version>
+        <version>1.4.8</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-data-schemaregistry</artifactId>
-        <version>1.3.8</version>
+        <version>1.3.9</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-data-schemaregistry-apacheavro</artifactId>
-        <version>1.1.8</version>
+        <version>1.1.9</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-data-tables</artifactId>
-        <version>12.3.13</version>
+        <version>12.3.14</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-developer-loadtesting</artifactId>
-        <version>1.0.4</version>
+        <version>1.0.5</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-digitaltwins-core</artifactId>
-        <version>1.3.11</version>
+        <version>1.3.12</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-identity</artifactId>
-        <version>1.9.2</version>
+        <version>1.10.0</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-identity-extensions</artifactId>
-        <version>1.1.6</version>
+        <version>1.1.7</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-iot-deviceupdate</artifactId>
-        <version>1.0.9</version>
+        <version>1.0.10</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
```

Comment:
There's [a new version](https://search.maven.org/artifact/com.azure/azure-json/1.1.0/jar) of `azure-json` (`1.1.0`). I wonder why the script did not update it.